EventsExpress/ClientApp/src/components/helpers/validators/email-field-validator.js

```diff
@@ -0,0 +1,13 @@
+export const emailField = values => {
```

Comment:
let pass here a single email value instead of a whole object
and do not check for requiring here

Reply:
this one can be probably 'email-address-validator(email)' or 'isValidEmail(email)'